x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/prepackaged_rules/windows_rare_user_type10_remote_login.json

```diff
@@ -20,5 +20,6 @@
     "Windows"
   ],
   "type": "machine_learning",
+  "note": "### Investigating an Unusual Windows User ###\nThis signal indicates activity for a rare and unusual Windows RDP (remote desktop) user. Here are some possible avenues of investigation:\n- Consider the user as identified by the username field. Is the user part of a group who normally logs into Windows hosts using RDP (remote desktop protocol)? Is this logon activity part of an expected workflow for the user? \n- Consider the source of the login. If the source is remote, could this be related to occasional troubleshooting or support activity by a vendor or an employee working remotely?",
```

Comment:
nit: May want `rule` instead of `signal` here, or something like `Signals from this rule indicate...` since this is only viewable on the Rule Details Page. Up to you, no chance necessary.

Reply:
OK we decided to make this change.